### PR TITLE
issue #868 dismiss registered field

### PR DESCRIPTION
**Pull request by IvanPizhenko**
Not sure if this is completely correct. See my comment in the #868.  Now just want to see if semaphore passes.

### CODE REVIEW

**Comment by IvanPizhenko** [comment]
Converting from draft to regular PR - let's anyway review it in this state.

File: FlowCrypt/src/androidTest/java/com/flowcrypt/email/ui/activity/enterprise/SignInActivityEnterpriseTest.kt [hunk 1]
```diff
@@ -135,16 +135,16 @@ class SignInActivityEnterpriseTest : BaseSignActivityTest() {
...
-                .setBody(gson.toJson(LoginResponse(null, isRegistered = true, isVerified = false)))
+                .setBody(gson.toJson(LoginResponse(null, isVerified = false)))
 
             EMAIL_DOMAIN_RULES_ERROR -> return MockResponse().setResponseCode(200)
-                .setBody(gson.toJson(LoginResponse(null, isRegistered = true, isVerified = true)))
+                .setBody(gson.toJson(LoginResponse(null, isVerified = true)))
 
             EMAIL_WITH_NO_PRV_CREATE_RULE -> return MockResponse().setResponseCode(200)
-                .setBody(gson.toJson(LoginResponse(null, isRegistered = true, isVerified = true)))
+                .setBody(gson.toJson(LoginResponse(null, FisVerified = true)))
```
Comment (DenBond7) [inline FlowCrypt/src/androidTest/java/com/flowcrypt/email/ui/activity/enterprise/SignInActivityEnterpriseTest.kt:147]: @IvanPizhenko It seems here is an issue, should be isVerified

**Comment by DenBond7** [comment]
@IvanPizhenko 
> Now just want to see if semaphore passes.

It was compilation issues. To prevent such an issue you can use 2 ways (for both way you have to have at least 1 connected device - an emulator or a real device):

1. Use Android Studio to just print tests names(**Print all tests**)
image

If that task passes there are no compilation issues

2. Use the following command `./gradlew clean :FlowCrypt:connectedDevTestDebugAndroidTest -Pandroid.testInstrumentationRunnerArguments.log=true`. If the task fails you will see compilation errors like this
image

File: FlowCrypt/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -379,8 +379,7 @@
     <string name="your_email_address">Your email address</string>
     <string name="pending">Pending&#8230;</string>
     <string name="mark_unread">Mark unread</string>
-    <string name="user_not_verified">User not verified</string>
-    <string name="user_not_registered_not_verified">User not registered and not verified</string>
+    <string name="user_not_registered_or_verified">User is not registered or not verified</string>
```
Comment (DenBond7) [inline FlowCrypt/src/main/res/values/strings.xml:382]: @tomholub Please look at this line. If we are going to drop `isRegistered` should we show `User is not registered or not verified` explanation? Maybe it should be just `User not verified`?

Comment (tomholub) [reply FlowCrypt/src/main/res/values/strings.xml:382]: Correct, just "User not verified successfully"

Comment (DenBond7) [reply FlowCrypt/src/main/res/values/strings.xml:382]: Got it. I'm going to fix it right now

**Review by DenBond7 — APPROVED** [review]

**Comment by IvanPizhenko** [comment]
@DenBond7 thanks! I just didn't know how to run all the tests in the Android project. Sure, I will do this next times.

**Comment by DenBond7** [comment]
@IvanPizhenko 
> @DenBond7 thanks! I just didn't know how to run all the tests in the Android project. Sure, I will do this next times.

Just want to clarify. My examples just print tests and don't run them. But it'll be enough to catch compilation errors

**Comment by IvanPizhenko** [comment]
ok, so you were meaning that this will not necessarily run test (due to various reasons like missing running AVD), but will try to compile everything and thus will catch compilation errors, right?

**Comment by DenBond7** [comment]
Almost. Unfortunately, any of my examples will not complete if you don't have at least one connected device. For example, `Android Studio` will not start that task if you don't have at least one connected device. `Gradle` will start a task but will fail if the compilation will pass but you don't have at least one connected device.

I'm thinking about that. Need to check all `Gradle` tasks. Maybe I'll find just a compilation task.

**Comment by IvanPizhenko** [comment]
assemble is standard gradle task for compilation only.

**Comment by IvanPizhenko** [comment]
@DenBond7 No, it eventually fails:
```shell
$ ./gradlew clean assemble

.....

> Task :FlowCrypt:compileConsumerReleaseKotlin FAILED
e: /home/ivan/pro/flowcrypt/flowcrypt-android/FlowCrypt/src/main/java/com/flowcrypt/email/FlowCryptApplication.kt: (27, 8): Unresolved reference: leakcanary
e: /home/ivan/pro/flowcrypt/flowcrypt-android/FlowCrypt/src/main/java/com/flowcrypt/email/FlowCryptApplication.kt: (123, 7): Unresolved reference: LeakCanary
e: /home/ivan/pro/flowcrypt/flowcrypt-android/FlowCrypt/src/main/java/com/flowcrypt/email/FlowCryptApplication.kt: (123, 18): Variable expected
e: /home/ivan/pro/flowcrypt/flowcrypt-android/FlowCrypt/src/main/java/com/flowcrypt/email/FlowCryptApplication.kt: (123, 27): Unresolved reference: LeakCanary
e: /home/ivan/pro/flowcrypt/flowcrypt-android/FlowCrypt/src/main/java/com/flowcrypt/email/FlowCryptApplication.kt: (124, 7): Unresolved reference: LeakCanary

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':FlowCrypt:compileConsumerReleaseKotlin'.
> Compilation error. See log for more details

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 3m 4s
67 actionable tasks: 30 executed, 37 from cache
```

**Comment by DenBond7** [comment]
@IvanPizhenko 
> Maybe I'll find just a compilation task.

I've found. You can use this one. It doesn't require a device.

`./gradlew clean :FlowCrypt:assembleDevTestDebugAndroidTest`

**Comment by DenBond7** [comment]
@IvanPizhenko 
Actually, it'll be better to use the following command. It covers Android and JUnit tests compilation.

`./gradlew clean :FlowCrypt:assembleDevTestDebugAndroidTest :FlowCrypt:testDevTestDebugUnitTest`

**Comment by IvanPizhenko** [comment]
@DenBond7 thanks! Just one note about "clean": it cleans the whole build output and rebuilds it again. If you are sure you are not messed up something, which is typical case, clean often can be omitted. However, if something constantly goes wrong, and looks like you have old code working instead of new, clean is the right thing to do. Otherwise incremental compilation of the only changed stuff is faster.

**Comment by IvanPizhenko** [comment]
@DenBond7 what is the difference between targets `:FlowCrypt:testDevTestDebugUnitTest` and `:FlowCrypt:connectedDevTestDebugAndroidTest` ? Just from the name I can assume that the former doesn't require emulator running while the latter requires. But that's just an assumption.

**Comment by IvanPizhenko** [comment]
Just now found out from `./gradlew tasks --all`:
```
FlowCrypt:connectedDevTestDebugAndroidTest - Installs and runs the tests for devTestDebug on connected devices.
FlowCrypt:testDevTestDebugUnitTest - Run unit tests for the devTestDebug build.
```

**Pull request merged**